fenix/app/src/main/res/values/strings.xml

```diff
@@ -1379,22 +1379,28 @@
     <string name="etp_settings">Protection Settings</string>
     <!-- Preference title for enhanced tracking protection settings -->
     <string name="preference_enhanced_tracking_protection">Enhanced Tracking Protection</string>
-    <!-- Title for the description of enhanced tracking protection -->
-    <string name="preference_enhanced_tracking_protection_explanation_title">Browse without being followed</string>
+    <!-- Preference summary for enhanced tracking protection settings -->
+    <string name="preference_enhanced_tracking_protection_summary">Now featuring Total Cookie Protection, our most powerful barrier yet against cross-site trackers.</string>
     <!-- Description of enhanced tracking protection. The first parameter is the name of the application (For example: Fenix) -->
-    <string name="preference_enhanced_tracking_protection_explanation">Keep your data to yourself. %s protects you from many of the most common trackers that follow what you do online.</string>
+    <string name="preference_enhanced_tracking_protection_explanation" moz:removedIn="112" tools:ignore="UnusedResources">Keep your data to yourself. %s protects you from many of the most common trackers that follow what you do online.</string>
+    <!-- Description of enhanced tracking protection -->
+    <string name="preference_enhanced_tracking_protection_explanation_2">Firefox protects you from many of the most common trackers that follow what you do online.</string>
```

Comment:
Driveby: The application name should not appear in these strings to prevent the brand being localized.
We should continue to use the values from the static strings 
https://github.com/mozilla-mobile/firefox-android/blob/6ab8322422d4cf35a2660bdefa583db2be1e9a92/fenix/app/src/main/res/values/static_strings.xml#L7-L8
and ensure localizers have enough context - through the string documentation to understand how to position the app name.
I see that previously we used `app_name` which would give `Firefox Nightly` for example. If it's unclear what brand to use I recommend syncing with UX to choose.

Reply:
Gotcha, yep that makes sense, thanks 👍 Have reached out to UX, but while waiting for confirmation I'll update to use "firefox" static

Reply:
Update: UX have confirmed this should use the `app_name` as before